OutOfSchool/OutOfSchool.Tests/OrganizationServiceTest.cs

```diff
@@ -0,0 +1,336 @@
+                .UseQueryTrackingBehavior(QueryTrackingBehavior.NoTracking)
+                .Options;
+
+            _context = new OutOfSchoolDbContext(dbContextOptions);
+
+            organizationRepository = new OrganizationRepository(_context);
+            _organizationService = new OrganizationService(organizationRepository, mapper);
+        }
+       
+
+        [OneTimeTearDown]
+        public void TearDown()
+        {
+            _context.Database.EnsureDeleted();
+        }
+
+        [Test,Order(1)]
+        public async Task Create_Organization_ReturnsCreatedOrganization()
+        {
+            // Arrange
+            var newOrganization = new OrganizationDTO
+            {
+                Id = 0b101,
+                Title = "Title",
+                Website = "Website",
+                Facebook = "Facebook",
+                Instagram = "Instagram",
+                Description = "Description",
+                MFO = "123456",
+                EDRPOU = "12345678",
+                INPP = "1234567891",              
+                Type = OrganizationType.FOP,
+                UserId = "123"
+            };
+
+            //Act
+            var organization = await _organizationService.Create(newOrganization).ConfigureAwait(false);
+           
+            //Assert
+            Assert.That(newOrganization.Id, Is.EqualTo(organization.Id), "Id's are equal");
+            Assert.That(newOrganization.Title, Is.EqualTo(organization.Title), "Titles are equal");                   
+        }
+
+        [Test, Order(3)]
+        public void Create_Organization_ReturnsArgumentNullException()
+        {
+            // Arrange
+            OrganizationDTO organization = null;
+
+            // Act and Assert
+            Assert.ThrowsAsync<ArgumentNullException>(
+                async () => await _organizationService.Create(organization).ConfigureAwait(false));
+        }
+
+        [Test, Order(4)]
+        public void Create_NotUniqueOrganization_ReturnsArgumentException()
+        {
+            // Arrange
+            var newOrganization = new OrganizationDTO
+            {
+                Id = 10,
+                Title = "bla",
+                Website = "Website",
+                Facebook = "Facebook",
+                Instagram = "Instagram",
+                Description = "Description",
+                MFO = "123456",
+                EDRPOU = "12345678",
+                INPP = "1234567891",
+                Type = OrganizationType.TOV,
+                UserId = "123"
+            };
+
+            // Act and Assert
+            Assert.ThrowsAsync<ArgumentException>(
+                async () => await _organizationService.Create(newOrganization).ConfigureAwait(false));           
+        }
+
+        [Test, Order(5)]
+        public async Task GetAll_Organizations_ReturnsSameAmountOrganizations()
+        {
+            // Arrange
+            var allOrganizations = await _organizationService.GetAll();
```

Comment:
service method is called, so this looks more like the Act step, not the Arrange step